src/Serilog.Sinks.RavenDB/LoggerConfigurationRavenDBExtensions.cs

```diff
@@ -42,14 +45,17 @@ public static class LoggerConfigurationRavenDBExtensions
             LogEventLevel restrictedToMinimumLevel = LevelAlias.Minimum,
             int batchPostingLimit = RavenDBSink.DefaultBatchPostingLimit,
             TimeSpan? period = null,
-            IFormatProvider formatProvider = null)
+            IFormatProvider formatProvider = null,
+            string defaultDatabase = null,
+            TimeSpan? expirationTimeSpan = null,
```

Comment:
Perhaps just `expiration` and `errorExpiration`? (Keeps consistent with `period` above, which would otherwise be `periodTimeSpan`).